websites/S/Shikimori/presence.ts

```diff
@@ -3,152 +3,208 @@ const presence = new Presence({
 });
 
 presence.on("UpdateData", async () => {
-	const presenceData: PresenceData = {
-			details: "Где-то на сайте",
+	const [privacy, logo, buttons] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("logo"),
+			presence.getSetting<boolean>("buttons"),
+		]),
+		presenceData: PresenceData = {
 			largeImageKey: "shikimori_logo",
```

Comment:
Imgur links are now preferred over Discord asset names.
```suggestion
			largeImageKey: "https://i.imgur.com/7tlc7or.png",
```